src/sdam/topology.ts

```diff
@@ -467,7 +478,11 @@ export class Topology extends TypedEventEmitter<TopologyEvents> {
 
       const skipPingOnConnect = this.s.options[Symbol.for('@@mdb.skipPingOnConnect')] === true;
       if (!skipPingOnConnect && server && this.s.credentials) {
-        await server.command(ns('admin.$cmd'), { ping: 1 }, {});
+        await server.command(
+          ns('admin.$cmd'),
+          { ping: 1 },
+          { operationTimeout: timeout, serverSelectionTimeout }
```

Comment:
Let's remove the `operationTimeout` from here and from `commandOptions` (relic of CSOT command execution ticket (NODE-5682) which not be done in this ticket.

Reply:
Decided to rename the `timeout` option on `AbstractOperation` to `operationTimeout` to make clear it's referring to the time out for the entire operation. This avoids the name conflict with the find options in future and clarifies the intention behind the option/field name.